ksqldb-docker/README.md

```diff
@@ -26,3 +27,14 @@ To build a new image with local changes:
     ```
     placeholder/confluentinc/ksqldb-docker       local.build   94210cd14384   About an hour ago   716MB
     ```
+   
+1. To use with the `docker-compose.yml` file in the root of the project, run the following additional commands to create
```

Comment:
can we avoid this step by having people just use the `ksqldb-docker` image in their compose files? that was part of the point of consolidating the image

Reply:
It's not their docker file - it's the one in the root of the project, and it needs to work with published images, which currently have two names.

Reply:
ack